examples/http/kubernetes_scheduler.json

```diff
@@ -0,0 +1,35 @@
+{
+    "logo": "kubernetes.svg",
+    "version": "1.0.0",
+    "name": "Kubernetes Scheduler",
+    "description": "This script is able to extract information about the Kubernetes scheduler (such as CPU consumption, fds open and max, memory usage, etc)",
```

Comment:
This is just personal preference but I'd change "This script is able to extract" -> "This script extracts"; no need to be too verbose